tapdb/sqlc/queries/assets.sql

```diff
@@ -155,13 +155,21 @@ RETURNING group_id;
 
 -- name: UpsertAssetGroupSig :one
 INSERT INTO asset_group_sigs (
-    genesis_sig, gen_asset_id, group_key_id
+    witness_id, gen_asset_id, group_key_id
 ) VALUES (
     $1, $2, $3
 ) ON CONFLICT (gen_asset_id)
     DO UPDATE SET gen_asset_id = EXCLUDED.gen_asset_id
 RETURNING sig_id;
 
+-- name: UpsertAssetGroupWitness :one
+INSERT INTO asset_group_witnesses (
```

Comment:
This isn't really an upsert, as there is nothing to conflict on here. So it will always insert a new record...